lib/internal/HtmlLinkParser.js

```diff
@@ -339,7 +339,7 @@ function reset(instance)
 	_.linkElementStack.length = 0;
 	
 	// Per-link variables
-	_.links.length = 0;
+	_.links = [];
```

Comment:
You put the comment in the constructor instead of within `reset()` :stuck_out_tongue_closed_eyes: